Assignment4/PointGrid.py

```diff
@@ -0,0 +1,170 @@
+class Grid:
+
+    def __init__(self, grid):
+        """
+        For grid initialization with a grid with elements.
+        :param grid: list(list(any))
+        :return: None
+        """
+        self._grid = grid
+        self._points = {}

```

Comment:
This would have been my first reaction too but since all the points will be created it could as well be a `list(list(Point()))`. And at that point, I suppose it could also be a `list(list(is_visited))` which would allow removing the `Point()` class altogether and simplifying the code? What do you think?